README.md

```diff
@@ -163,8 +188,21 @@ const  address  =  await FNSInstance.getAddress('juan.fil', BlockchainIdentifier
 const  address  =  await FNSInstance.getAddress('juan.fil', BlockchainIdentifier.DOGECOIN)
 
 
-/* Normal content fetching */
-const contentHash = await FNSInstance.getContent('juan.fil')
+/* Normal content fetching 
+To get content hash attached to a fns name
+*/
+
+const  contentHash  =  await FNSInstance.getContent('juan.fil')
```

Comment:
Apart from this, show how to get the name from address using FNSLib and describe the flow of how to use it